### PR TITLE
fixes issue #4

### PR DESCRIPTION
**Pull request by haydencordeiro**

### CODE REVIEW

File: tracer/main.py [hunk 1]
```diff
@@ -3,12 +3,13 @@
 import re
 import socket
 import sys
-
 import click
 import requests
 from colorama import Fore as fg
 from colorama import Style as sty
 from tld import get_fld
+import sys
```
Comment (danielmichaels) [inline tracer/main.py:11]: This is already imported on line 5.

File: tracer/main.py [hunk 2]
```diff
@@ -22,18 +23,28 @@
...
+    help="File location where you want to save the Output",
+)
+def main(url, full,output):
+    
     """
     HTTP-Tracer returns the redirects on way to the destination URL.
 
     If user does not supply 'https://' it will default to 'http://'
     """
+    if(output):
```
Comment (danielmichaels) [inline tracer/main.py:39]: In python, for a single `if` you do not need parenthesis. 

Do this instead

```python
if output:
```

File: tracer/main.py [hunk 3]
```diff
@@ -22,18 +23,28 @@
...
+        sys.stdout = open(output[0], 'w')
```
Comment (danielmichaels) [inline tracer/main.py:40]: I can see what you're doing here but why is it `output[0]`. It's not immediately apparent what `output[0]` is both as a data structure and its implementation.

Is `output[0]` a string, or list?

**Review by danielmichaels — CHANGES_REQUESTED** [review]
Thanks for the PR.

I have added some comments to the code and things to fix up - nothing major.

As a rule, whenever I raise a PR with another project, I like to include a detailed description of the problem I am solving with links to any issues. In this case, a segue to our conversation in #4 would have been a great starter. This is especially important on projects where multiple maintainers are present - the person reviewing this may not know of the conversation in the issue tracker. Lastly, an empty comment field on a bigger project is a pretty quick way to get your PR ignored - without context as to why this exists many maintainers simply do not have the time to reverse engineer the *why*.

File: tracer/main.py (hunk 1 above)
Comment (haydencordeiro) [reply tracer/main.py:11]: Really sorry will look into it.

**Review by danielmichaels — APPROVED** [review]

**Comment by danielmichaels** [comment]
Thank you for your contribution.

**Pull request merged**